gamebook/terminal_output.py

```diff
@@ -0,0 +1,6 @@
+from gamebook.output import Output
+
+class TerminalOutput(Output):
+    
+    def output(self,content):
```

Comment:
try to lint all of your files upon saving: https://code.visualstudio.com/docs/python/linting

Reply:
i do lint after every save. i don't get any errors here

Reply:
I meant beautify the code (you have some spacing problems).